### ISSUE TITLE
decrypt --session-key-out does not write the session key in the expected format

### ISSUE THREAD

**Issue by dkg** [pgpainless/sop-java#13]
This should create an encrypted message `test.msg` and extract the session key of that message to a file named `session.key`:

```
pgpainless-cli generate-key test > test.key
pgpainless-cli extract-cert < test.key > test.cert
echo test | pgpainless-cli encrypt test.cert > test.msg
pgpainless-cli --session-key-out session.key test.key < test.msg
```

The resulting file should be decryptable even after destroying `test.key` using:

```
pgpainless-cli decrypt --with-session-key session.key < test.msg
```

however, instead, i get:

```
sop.exception.SOPGPException$PasswordNotHumanReadable
	at sop.util.UTF8Util.decodeUTF8(UTF8Util.java:37)
	at sop.cli.picocli.commands.AbstractSopCmd.stringFromInputStream(AbstractSopCmd.java:205)
	at sop.cli.picocli.commands.DecryptCmd.setWithSessionKeys(DecryptCmd.java:177)
	at sop.cli.picocli.commands.DecryptCmd.run(DecryptCmd.java:91)
	at picocli.CommandLine.executeUserObject(CommandLine.java:1939)
	at picocli.CommandLine.access$1300(CommandLine.java:145)
	at picocli.CommandLine$RunLast.executeUserObjectOfLastSubcommandWithSameParent(CommandLine.java:2358)
	at picocli.CommandLine$RunLast.handle(CommandLine.java:2352)
	at picocli.CommandLine$RunLast.handle(CommandLine.java:2314)
	at picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:2179)
	at picocli.CommandLine$RunLast.execute(CommandLine.java:2316)
	at picocli.CommandLine.execute(CommandLine.java:2078)
	at sop.cli.picocli.SopCLI.execute(SopCLI.java:84)
	at org.pgpainless.cli.PGPainlessCLI.execute(PGPainlessCLI.java:29)
	at org.pgpainless.cli.PGPainlessCLI.main(PGPainlessCLI.java:22)
```

If i manually convert the file to be in the right format, it does work.

### version info

```
$ pgpainless-cli version --extended
PGPainless-SOP 1.3.7
https://codeberg.org/PGPainless/pgpainless/src/branch/master/pgpainless-sop

Implementation of the Stateless OpenPGP Protocol Version 04
https://datatracker.ietf.org/doc/html/draft-dkg-openpgp-stateless-cli-04

Based on pgpainless-core 1.3.7
https://pgpainless.org

Using Bouncy Castle 1.71
https://www.bouncycastle.org/java.html
```

**Comment by vanitasvitae** [comment]
I guess in the first paragraph
```
pgpainless-cli --session-key-out session.key test.key < test.msg
```
should be 
```
pgpainless-cli decrypt --session-key-out session.key test.key < test.msg
```
right?

Anyway, I wonder when exporting out the session key broke. I could swear it worked at some point. Thanks for reporting!

**Comment by vanitasvitae** [comment]
Fixed in dd5d790e211639c063a5019324f21e31d0769d54
